autoupdate.services/src/org/netbeans/modules/autoupdate/updateprovider/AutoupdateCatalogCache.java

```diff
@@ -155,24 +156,24 @@ public void storeLicense(String name, String content) {
     
     private String readLicenseFile(String name) {
         File file = getLicenseFile(name);
-        FileInputStream fr = null;
+        InputStream is = null;
```

Comment:
Why not preserve the `fr` name?

Reply:
Because the "fr" implies FileReader, which it is not. The fr naming is not really consistent with the naming in the rest of the modules I made changes at.